Source/Core/Core/Boot/Boot.cpp

```diff
@@ -226,7 +226,11 @@ bool CBoot::BootUp()
 	NOTICE_LOG(BOOT, "Booting %s", _StartupPara.m_strFilename.c_str());
 
 	g_symbolDB.Clear();
-	VideoInterface::Preset(_StartupPara.bNTSC);
+
+	// PAL Wii uses NTSC framerate and linecount in 60Hz/480i and 60Hz/480p modes
+	const bool bPAL60 = _StartupPara.bWii && (SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.E60") || SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.PGS"));
```

Comment:
Should IPL.PGS really be checked here? It makes more sense to only have IPL.E60 control whether 60 Hz is used. The only reason to check IPL.PGS that I can think of is to prevent users from setting the 576p mode, but if we want to do that, we should prevent users from setting that flag combination in SYSCONF instead of only blocking it here.